### ISSUE TITLE
Hack the Space 14th of December

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#1966]
Hey all,

During the [community meeting yesterday](https://groups.google.com/d/msgid/does-liverpool/68679743-9138-4D47-93F8-F7588385C14B%40zarino.co.uk?utm_medium=email&utm_source=footer) we decided we need another Hack the Space day. We thought it would be easiest if we roll it into a Makerday so people can kill two birds with one stone, so to speak.

The next Makerday is on Saturday the 14th of December from noon till six - we're asking people who want to help out with hacking the space to come from 10, so we can get the bigger jobs done before people arrive for Makerday.

The main jobs we identified for now are:

- tidy up the machine room where the CNC is,
- make some progress on the PAT testing,
- Jackie would like to replace the desks in the pen plotter corner with the grey desks stored in the event space, which should be more stable.

Obviously there is more to do, but those are the primary ones we're hoping to focus on.

Hope to see you there?

Cheers,

Snoof

**Comment by amcewen** [comment]
There was a Hack the Space on 14th December.  Not massively attended sadly, but it was December and we didn't send any reminders out.

@magman2112 and @Andygwk got a bunch of PAT testing done; I ended up triaging and tidying some bugs
